src/main/kotlin/org/rust/ide/refactoring/move/common/RsMoveCommonProcessor.kt

```diff
@@ -161,8 +161,9 @@ class RsMoveCommonProcessor(
                     traitMethodsProcessor.preprocessInsideReferencesToTraitMethods(conflicts, elementsToMove)
 
                     if (!isUnitTestMode) {
-                        ProgressManager.getInstance().progressIndicator.text = message("detecting.possible.conflicts")
+                        ProgressManager.getInstance().progressIndicator.text = "Detecting possible conflicts..."
```

Comment:
why?

Reply:
<img width="901" alt="Screenshot 2021-03-24 at 19 22 57" src="https://user-images.githubusercontent.com/6079006/112345762-61679380-8cd6-11eb-8a60-01223525fe17.png">

Reply:
Looks like all usages in the platform have a similar issue. I suggest not to touch it currently